xdggs/accessor.py

```diff
@@ -50,6 +50,11 @@ def coord(self) -> xr.DataArray:
             )
         return self._obj[self._name]
 
+    @property
+    def params(self) -> dict:
```

Comment:
I'm wondering if we shouldn't expose `DGGSInfo` objects publicly and return such an object here instead of a dict? E.g.,

```python
    @property
    def grid_info(self) -> DGGSInfo:
        return self.index.grid_info
```

- This would make the API more consistent xdggs-wise
- Options can be better documented
- If we want a dict, just call `.dggs.grid_info.to_dict()`

Reply:
I added this because I noticed that it is very useful to have access to the normalized grid parameters. While `.dggs.grid_info.to_dict()` certainly works, it is quite a lot to write, especially if you need to access it from multiple places.